recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ build:
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
   entry_points:
-    - nwbinspector = nwbinspector.nwbinspector:inspect_all_cli
+    - nwbinspector = nwbinspector._inspection_cli:_inspect_all_cli
```

Comment:
Ah; I had completely forgotten that the feedstock defined its own entrypoints

Great catch @stephprince!

This will likely still be the case even after we swap to using `pyproject.toml` for building the project, correct?

Reply:
I also had updated the dependencies to match the `requirements.txt`, but looks like that's causing a failure with hdmf-zarr.

Reply:
> I also think [src/nwbinspector/nwbinspector/__init__.py](https://github.com/NeurodataWithoutBorders/nwbinspector/blob/f5bfe8280dbfceda76acb22fa85cfd596bae0bed/src/nwbinspector/nwbinspector/__init__.py#L12-L14) should be updated so that the InspectorOutputJSONEncoder is imported from .._formatting instead of .._configuration if we want to keep it as soft deprecation

Cool, can you send a quick PR for that and we can cut a quick follow-up release?

Reply:
will do!